### ISSUE TITLE
How to use in VS Project with ASP.NET Core .NET Core

### ISSUE THREAD

**Issue by guehmann** [metavertex/MetaVertex.DataModel#1]
Hello, 

have you any samples how to implement in an project. Actually i use IBM.Data.DB2.iSeries Assembly which does not work with .NET Standard. I'd like to implement your assembly.

Don't know how to initialize a database connection with your assembly.

Perhaps you can help me.
Thanks in advance.

Jens.

**Comment by irontoby** [comment]
Hi Jens, this project does not help at all with connecting to the database, only converting the objects from an existing connection. It was a quick experiment that's no longer maintained; I recommend something like Dapper instead.

I tried many ways to use the `IBM.Data.DB2.iSeries` assembly from .NET Standard / Core and was not successful. There is at least one method that's not implemented in Core or the Windows compatibility pack. Depending on your needs you may be able to go the ODBC route.

**Comment by guehmann** [comment]
Hello Toby,
thanks for your reply. 

You are sure. The is no simple way for this. So I decided and go back to  iSeries with OLEDB with an origin .NET Framework API Projekt with mit ASP.NET Core Front End. (It's a little bit faster than ODBC).
I got no Information from IBM how to get an Licence File for using EntityFrameworkCore for iSeries, it would be much faster an easier to use with ASP.NET Core.

Thank you a lot.
Best regards Jens.
